botorch_community/acquisition/input_constructors.py

```diff
@@ -0,0 +1,148 @@
+#!/usr/bin/env python3
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+#
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+r"""
+A registry of helpers for generating inputs to acquisition function
+constructors programmatically from a consistent input format.
+"""
+
+from __future__ import annotations
+
+from typing import Any, Callable, Dict, List, Optional, Tuple, Type
+
+import torch
+from botorch.acquisition.acquisition import AcquisitionFunction
+from botorch.acquisition.input_constructors import allow_only_specific_variable_kwargs
+from botorch.acquisition.utils import get_optimal_samples
+from botorch.models.model import Model
+from botorch_community.acquisition.bayesian_active_learning import (
+    qBayesianActiveLearningByDisagreement,
+    qBayesianQueryByComittee,
+    qBayesianVarianceReduction,
+    qStatisticalDistanceActiveLearning,
+)
+from botorch_community.acquisition.scorebo import qSelfCorrectingBayesianOptimization
+from torch import Tensor
+
+
+COMMUNITY_ACQF_INPUT_CONSTRUCTOR_REGISTRY = {}
```

Comment:
We can probably just add into the main registry in here, which would happen only when this file is imported. That'd make it easier to use these constructors from Ax. This doesn't need to happen in this PR though.

Reply:
I remember trying that, but I'm not sure it worked. I'm happy to work on that, but should I leave this unchanged for now?